src/csvcubed/models/cube/qb/catalog.py

```diff
@@ -3,25 +3,17 @@
 -----------------------
 """
 import json
-import logging
 from dataclasses import dataclass, field
 from datetime import date, datetime, time
 from pathlib import Path
-from typing import Dict, List, Optional, Union
+from typing import Dict, Optional, Union
 
 from csvcubedmodels.rdf import dcat
 
 from csvcubed.models.cube.catalog import CatalogMetadataBase
 from csvcubed.models.uriidentifiable import UriIdentifiable
-from csvcubed.models.validatedmodel import ValidatedModel, ValidationFunction
-from csvcubed.models.validationerror import ValidateModelPropertiesError
+from csvcubed.models.validatedmodel import ValidationFunction
 from csvcubed.utils import validations as v
-from csvcubed.utils.validators.uri import validate_uri as pydantic_validate_uri
-from csvcubed.utils.validators.uri import (
-    validate_uris_in_list as pydantic_validate_uris_in_list,
-)
-
-_logger = logging.getLogger(__name__)
```

Comment:
Why's the logger disappearsed? Isn't it used?

Reply:
If it's not referenced, it isn't doing anything so it isn't necessary. I'm just surprised it was in there if it wasn't necessary.